.github/workflows/isotovideo.yml

```diff
@@ -1,11 +1,43 @@
 ---
 name: isotovideo
 # yamllint disable-line rule:truthy
-on: [push, pull_request]
+on:
+  - push
+  - pull_request
 jobs:
-  test:
-    runs-on: ubuntu-latest
+  isotovideo:
+    strategy:
+      matrix:
+        # An example matrix, it'll create a derivate job for each
+        # item specified here. For simplicity sake the example below
+        # has the full var line since it's easy enough.
+        #
+        # For more complex setups it'll be worth exploring advance matrix setups
+        # see: https://docs.github.com/en/actions/using-jobs/using-a-matrix-for-your-jobs#example-expanding-configurations
+        isotovideo-args:
+          - "--exit-status-from-test-results qemu_no_kvm=1 casedir=."
+    runs-on: ubuntu-20.04
```

Comment:
Is there a reason to run it on *older* ubuntu version?

Reply:
none whatsoever, I seem to recall `ubuntu-latest` was not the real latest when I started it. I'll throw in `ubuntu-latest` and see how it behaves